docker-compose.yml

```diff
@@ -7,35 +7,80 @@ services:
     volumes:
       - db_data:/var/lib/postgresql/data
     ports:
-      ## Expose the port for tooling (SQL language server in IDE, connecting with GUI's etc)
-      - '5432:5432'
+      - ${LOCAL_POSTGRES_PORT}:${LOCAL_POSTGRES_PORT}
     environment:
-      POSTGRES_PASSWORD: postgrespassword
+      POSTGRES_PASSWORD: ${LOCAL_POSTGRES_PASSWORD}
```

Comment:
https://docs.docker.com/compose/environment-variables/ seems to show strings using ENV_VAR interpolation requiring quotes... maybe it needs to be
`       POSTGRES_PASSWORD: "${LOCAL_POSTGRES_PASSWORD}"`
?
@exrhizo